smarts/core/scenario.py

```diff
@@ -715,6 +715,16 @@ def bubbles(self):
     def mission(self, agent_id):
         return self._missions.get(agent_id, None)
 
+    def _resolve_log_dir(self, log_dir):
+        home_dir = os.path.expanduser("~")
+        _, home_user = os.path.split(home_dir)
+        if log_dir is None:
+            log_dir = os.path.abspath(f"/tmp/smarts_{home_user}/_sumo_run_logs")
```

Comment:
`/tmp` is not something we can rely on to be cross platform.

Use `mkdtemp` with a `SMARTS_` prefix instead. https://docs.python.org/3.9/library/tempfile.html#tempfile.mkdtemp

Reply:
+1